src/main/resources/metadata/ncit.json

```diff
@@ -0,0 +1,161 @@
+{
```

Comment:
Why are we adding this file here? is it for testing?  Because this file lives in evsrestapi-operations and having it here is confusing.   We should add "extra subsets" to the *real* file in evsrestapi-operations.

Reply:
This is there because we talked about having the test for reading extraSubsets from the json file use a local version of the file instead of the operations version. I can change that back to the operations version without much trouble if that's better